crypto/benches/commit.rs

```diff
@@ -44,20 +44,16 @@ fn commit_add_n_clients_bench(c: &mut Criterion) {
                     || {
                         async_std::task::block_on(async {
                             let (central, id) = setup_mls(ciphersuite, credential.as_ref(), in_memory).await;
-                            let mut members = Vec::new();
+                            let mut kps = vec![];
```

Comment:
as we know `i` we might be able to use `Vec::with_capacity(i)` to use a tad less memory